bookwyrm/templates/settings/celery.html

```diff
@@ -116,6 +116,37 @@ <h3>{{ worker_name }}</h3>
 </div>
 {% endif %}
 
+<section class="block">
+    <details>
+        <summary><div class="content"><h2>{% trans "Clear Queues" %}</h2></div></summary>
```

Comment:
I believe the test is failing because you aren't supposed to nest a div inside a summary element

Reply:
Yeah, decided to just give up on making it collapsible, that's too fancy I guess :sweat_smile: